agent.go

```diff
@@ -116,7 +116,8 @@ func (a *Agent) loopSyncMonitors() {
 func (a *Agent) syncMonitors(init bool) error {
 	monitors, hash, err := a.retrieveMonitors()
 	if err != nil {
-		return fmt.Errorf("couldn't retrieve monitors from upstream, see: %v", err)
+		logrus.Errorf("couldn't retrieve monitors from upstream, see: %v", err)
+		return nil
```

Comment:
huh? Why  shouldn't we return the error?

Reply:
Because if we do the application quits.

Reply:
It then loops around to next operation, and tries again next time, so no potential issues here.